pkg/lib/operation/project/remote/file/download/operation.go

```diff
@@ -0,0 +1,96 @@
+// Package download contains the implementation of the "kbc project remote file download" command.
+// nolint: forbidigo
```

Comment:
Mohl bys tenhle `nolint` dát jen na ty místa, kde používáš file APIs z `os` package? Jen aby se tady v budoucnosti něco neztratilo.

Reply:
Jasně. 👍  Je to tam: https://github.com/keboola/keboola-as-code/pull/1210/commits/2d36eee952559ce9416f4bce4e3de3cec19160f5